soc/microchip/pic32cmmc/Kconfig.defconfig

```diff
@@ -7,6 +7,9 @@ if SOC_FAMILY_MICROCHIP_PIC32CMMC
 
 rsource "*/Kconfig.defconfig"
 
+config GPIO
+	default y
+
```

Comment:
why is this needed?

Reply:
I just copied from sam0.

Is forced-pushing acceptable here to make the changes?

Reply:
Yes, rebase, make changes, force push